modules/electromagnetics/doc/content/source/auxkernels/AzimuthMagneticTimeDerivScalar.md

```diff
@@ -0,0 +1,28 @@
+# AzimuthMagneticTimeDerivScalar
```

Comment:
what do you mean by Scalar here? It is as opposed to vector?

Reply:
Scalar here reference in the inputs being scalar components of the electric field, instead using a vector input of the electric field (which is what `AzimuthMagneticTimeDerivVector` is for).

Reply:
what about the outputs? are they the same?

Reply:
Yes, the outputs are the same (a scalar output of the the derivative of the azimuthal component of the magnetic field). Both test for `AzimuthMagneticTimeDerivScalar` and `AzimuthMagneticTimeDerivVector`uses the same manufactured solution. The only difference is the option to use different input forms for the electric field.

Reply:
I think I would be in favor of consolidating the two objects, so you can remove scalar/vector from the name. This makes it more clearer and less of a word soup
Simple logic in the constructor makes sure that the parameters passed are not overlapping

I think this not what the EM module has been doing so far though so maybe you should see with Casey if he disagrees

Reply:
Consolidated in [bf0cf21](https://github.com/idaholab/moose/pull/28759/commits/bf0cf213457cc381f4617eb0b0cfdbf513c9cf4f)